tests/unit/test_operator.py

```diff
@@ -40,6 +47,39 @@ def test_defaults(self, clickhouse_hook_mock: mock.MagicMock):
         )
         clickhouse_hook_mock().run.assert_called_once_with(sql, None)
 
+    def test_template_fields_overrides(self):
+        assert ClickHouseOperator.template_fields == ('_sql',)
+
+    def test_resolve_template_files_value(self):
+        with NamedTemporaryFile(suffix='.sql') as f:
+            f.write(b'{{ ds }}')
+            f.flush()
+            template_dir = os.path.dirname(f.name)
+            template_file = os.path.basename(f.name)
+
+            with DAG('test-dag', start_date=DEFAULT_DATE, template_searchpath=template_dir):
+                task = ClickHouseOperator(task_id='test_task', sql='SELECT 1')
+
+            task._sql = template_file
```

Comment:
Why do you override `task._sql` here and not simply set `sql` to `template_file` in the `ClickHouseOperator` constructor? I am not quite familiar with files functionality in Airflow, and I would assume `ClickHouseOperator(task_id='test_task', sql=template_file)` to work fine.